packages/fx-core/resource/package.nls.json

```diff
@@ -293,10 +293,12 @@
   "core.createProjectQuestion.projectType.officeAddin.framework.placeholder": "Select a framework",
   "core.createProjectQuestion.projectType.tab.detail": "Embed your own web content in Teams, Outlook, and the Microsoft 365 app",
   "core.createProjectQuestion.projectType.tab.title": "App Features Using a Tab",
+  "core.createProjectQuestion.projectType.copilotExtension.label": "Copilot Extension",
+  "core.createProjectQuestion.projectType.copilotExtension.title": "App Features using Copilot Extension",
+  "core.createProjectQuestion.projectType.copilotExtension.detail": "Create declarative copilot, API plugin, or both for customized guidance and integration.",
   "core.createProjectQuestion.projectType.copilotPlugin.detail": "Create a plugin to extend Microsoft Copilot for Microsoft 365 using your APIs",
   "core.createProjectQuestion.projectType.copilotPlugin.label": "API Plugin",
-  "core.createProjectQuestion.projectType.copilotPlugin.title": "API Plugin",
-  "core.createProjectQuestion.projectType.copilotPlugin.placeholder": "Select an option",
+  "core.createProjectQuestion.projectType.copilotExtension.placeholder": "Select an option",
   "core.createProjectQuestion.projectType.customCopilot.detail": "Build intelligent chatbot in Microsoft Teams easily using Teams AI Library",
```

Comment:
Revised: Build intelligent chatbot with Teams AI Library where you manage orchestration and provide your own LLM.

Note: Confirm with Junjie.

Reply:
@MuyangAmigo are you OK the revised string proposed by Supriya?

Reply:
Yes this was discussed in an other thread. Thanks folks!